src/resources/ethernet_esp32.cc

```diff
@@ -56,7 +56,7 @@ enum {
 const int kInvalidEthernet = -1;
 
 // Only allow one instance of WiFi running.
-ResourcePool<int, kInvalidEthernet> ethernet_pool(
+static ResourcePool<int, kInvalidEthernet> ethernet_pool(
   0
```

Comment:
2 or 4 space indent for this? Looks like we're not very consistent across the code base. I think we mostly use 4.